torch/csrc/distributed/rpc/tensorpipe_utils.cpp

```diff
@@ -2,7 +2,11 @@
 
 #ifdef USE_TENSORPIPE
 
-#include <torch/csrc/distributed/rpc/utils.h>
+#ifdef USE_CUDA_NOT_ROCM
+#include <tensorpipe/common/cuda_buffer.h>
```

Comment:
this file does not include `tensorpipe.h`. Should I include that instead?

Reply:
I'm surprised it doesn't include it already but, yes, sure, I think we should include that file instead. (That header is the only "public" interface we'd like to have, so that we can move/rename the other headers without breaking users)

Reply:
addressed